src/Microsoft.FeatureManagement/ConfigurationFeatureDefinitionProvider.cs

```diff
@@ -151,41 +155,41 @@ public async IAsyncEnumerable<FeatureDefinition> GetAllFeatureDefinitionsAsync()
             }
         }
 
-        private FeatureDefinition GetDotnetSchemaFeatureDefinition(string featureName)
+        private static FeatureDefinition GetDotnetSchemaFeatureDefinition(string featureName, IConfiguration configuration, bool rootConfigurationFallbackEnabled)
```

Comment:
The current implementation (i.e. setting schemaReadFunc in the constructor) don't require these methods to be static right? Doing this is for performance, is that true?

Reply:
True. I don't think there's even a performance gain with static here since we already have an instance of the class. Going to remove static from these.